tests.bzl

```diff
@@ -0,0 +1,552 @@
+    "mediasession",
+    "media-source",
+    "merchant-validation",
+    "mimesniff",
+    "mixed-content",
+    "mst-content-hint",
+    "navigation-api",
+    "navigation-timing",
+    "netinfo",
+    "network-error-logging",
+    "notifications",
+    "old-tests",
+    "orientation-event",
+    "orientation-sensor",
+    "page-lifecycle",
+    "page-visibility",
+    "paint-timing",
+    "parakeet",
+    "partitioned-popins",
+    "payment-handler",
+    "payment-method-basic-card",
+    "payment-method-id",
+    "payment-request",
+    "performance-timeline",
+    "periodic-background-sync",
+    "permissions",
+    "permissions-policy",
+    "permissions-request",
+    "permissions-revoke",
+    "picture-in-picture",
+    "png",
+    "pointerevents",
+    "pointerlock",
+    "preload",
+    "presentation-api",
+    "print",
+    "private-aggregation",
+    "private-click-measurement",
+    "proximity",
+    "push-api",
+    "quirks",
+    "referrer-policy",
+    "remote-playback",
+    "reporting",
+    "requestidlecallback",
+    "resize-observer",
+    "resources",
+    "resource-timing",
+    "sanitizer-api",
+    "savedata",
+    "scheduler",
+    "screen-capture",
+    "screen-details",
+    "screen-orientation",
+    "screen-wake-lock",
+    "scroll-animations",
+    "scroll-to-text-fragment",
+    "secure-contexts",
+    "secure-payment-confirmation",
+    "selection",
+    "serial",
+    "server-timing",
+    "service-workers",
+    "shadow-dom",
+    "shape-detection",
+    "shared-storage",
+    "shared-storage-selecturl-limit",
+    "signed-exchange",
+    "soft-navigation-heuristics",
+    "speculation-rules",
+    "speech-api",
+    "storage",
+    "storage-access-api",
+    "streams",
+    "subapps",
+    "subresource-integrity",
+    "svg",
+    "svg-aam",
+    "timing-entrytypes-registry",
+    "tools",
+    "top-level-storage-access-api",
+    "touch-events",
+    "trusted-types",
+    "trust-tokens",
+    "ua-client-hints",
+    "uievents",
+    "upgrade-insecure-requests",
+    "url",
+    "urlpattern",
+    "user-timing",
+    "vibration",
+    "video-rvfc",
+    "viewport",
+    "virtual-keyboard",
+    "visual-viewport",
+    "wai-aria",
+    "wasm",
+    "web-animations",
+    "webaudio",
+    "webauthn",
+    "web-bundle",
+    "webcodecs",
+    "WebCryptoAPI",
+    "webdriver",
+    "webgl",
+    "webgpu",
+    "webhid",
+    "webidl",
+    "web-locks",
+    "webmessaging",
+    "webmidi",
+    "web-nfc",
+    "webnn",
+    "web-otp",
+    "webrtc",
+    "webrtc-encoded-transform",
+    "webrtc-extensions",
+    "webrtc-ice",
+    "webrtc-identity",
+    "webrtc-priority",
+    "webrtc-stats",
+    "webrtc-svc",
+    "web-share",
+    "websockets",
+    "webstorage",
+    "webtransport",
+    "webusb",
+    "webvr",
+    "webvtt",
+    "webxr",
+    "window-management",
+    "workers",
+    "worklets",
+    "x-frame-options",
+    "xhr",
+]
+
+ACCELEROMETER = glob(["accelerometer/**/*"])
```

Comment:
also, rather than exporting each of them manually, maybe exporting a dictionary `TEST_GROUPS` will be better?

you can use dict expression to easily generate it: https://bazel.build/rules/lib/core/dict

Reply:
Fantastic recommendation. Thank you!